skeleton/aws/modules/cloudwatch/main.tf

```diff
@@ -1,5 +1,5 @@
 # tfsec:ignore:aws-cloudwatch-log-group-customer-key
 resource "aws_cloudwatch_log_group" "main" {
   name              = "awslogs-${var.namespace}-log-group"
-  retention_in_days = var.log_retention_in_days
+  retention_in_days = 14 # days
```

Comment:
I would not suggest hardcoding this value, it should be customizable by the environment to
- optimize the cost
- having enough data to debug when there is an incident.

e.g. for staging, 14 days would be enough, but prod should require 3 months of retention at least (in another project, we're updating the retention to 1 year for production environment).

Reply:
That makes sense, let me update it 🚀

Reply:
aa2324e ✅ 

I've added it to the code-project variables. As either we have it directly configurable in TerraformCloud, or we should use direct values. But we should not keep a variable that is never used and that would require some code changes to be used as a TerraformCloud variable 👍